.github/workflows/build.yaml

```diff
@@ -241,7 +241,22 @@ jobs:
     name: Lint
     runs-on: ubuntu-20.04
     steps:
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15
```

Comment:
We should keep using the same version of go. See also https://github.com/kubesphere/ks-devops/blob/6d9f1f47378be638360bad0f23177dcb409beaa2/go.mod#L3

Reply:
Got it. But how about this one?

https://github.com/kubesphere/ks-devops/blob/a4598c1bbc47d2e4b95d2a99a4f193ad9aad2e2c/.github/workflows/build.yaml#L217-L226

Reply:
That is not a reason to have multiple Golang versions. In another word, it's incorrect.

Reply:
I will fix the wrong Golang version in another pull request.